### PR TITLE
update predict endpoint

### PR DESCRIPTION
**Pull request by vietnguyengit**

### CODE REVIEW

File: tests/test_pipeline.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    pipeline(
+        isDataChanged=False,
+        usePretrainedModel=False,
+        description=item_description,
+        selected_model="development",
+    )
+
+
+if __name__ == "__main__":
+    test()
```
[the remark this answers is not in the document]
Comment (vietnguyengit) [reply tests/test_pipeline.py:19]: when you check the test, the proper command to do is `poetry run pytest`

you should resist the attention to do as with vanilla python where you normally try `python somefile.py` @shaunahu

[the remark this answers is not in the document]
Comment (shaunahu) [reply tests/test_pipeline.py:19]: No, I am not start on unit tests for functions. So I simply made a `test` function to try if the pipeline logic works. Can I remove this file from tests folder and work on the unit tests later? @vietnguyengit

[the remark this answers is not in the document]
Comment (vietnguyengit) [reply tests/test_pipeline.py:19]: thanks @shaunahu you can simply add `# TODO: .....` at the begining of the file.

[the remark this answers is not in the document]
Comment (shaunahu) [reply tests/test_pipeline.py:20]: @vietnguyengit Thank you. The TODO comment is added. I will work on the unit test from another branch (based on the main branch)

**Review by shaunahu — APPROVED** [review]

**Pull request merged**